OHHTTPStubs/OHHTTPStubsProject.xcconfig

```diff
@@ -0,0 +1,30 @@
+//
+//  OHHTTPStubsProject.xcconfig
+//  OHHTTPStubs
+//
+//  Created by Jeff Lett on 3/7/19.
+//  Copyright © 2019 AliSoftware. All rights reserved.
+//
+
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+// These build settings are used to disable specific unit tests.
+// They can and should be overridden when running tests in CI using an xcodebuild argument.
+
+
+//   xcodebuild Example:
+//     xcodebuild -workspace OHHTTPStubs/OHHTTPStubs.xcworkspace -scheme "OHHTTPStubs iOS StaticLib" -sdk iphonesimulator -configuration Debug ONLY_ACTIVE_ARCH=NO OHHTTPSTUBS_SKIP_TIMING_TESTS=1 -destination 'name=iPhone 7,OS=latest' clean build test
+//
+//   rake Example:
+//     rake ios['iOS StaticLib','latest','build-for-testing test-without-building',"OHHTTPSTUBS_SKIP_TIMING_TESTS=1"]
+OHHTTPSTUBS_SKIP_TIMING_TESTS=0
+
+//   xcodebuild Example:
+//     xcodebuild -workspace OHHTTPStubs/OHHTTPStubs.xcworkspace -scheme "OHHTTPStubs iOS StaticLib" -sdk iphonesimulator -configuration Debug ONLY_ACTIVE_ARCH=NO OHHTTPSTUBS_SKIP_REDIRECT_TESTS=1 -destination 'name=iPhone 7,OS=latest' clean build test
+//
+//   rake Example:
+//     rake ios['iOS StaticLib','latest','build-for-testing test-without-building',"OHHTTPSTUBS_SKIP_REDIRECT_TESTS=1"]
+OHHTTPSTUBS_SKIP_REDIRECT_TESTS=0
+
+GCC_PREPROCESSOR_DEFINITIONS=OHHTTPSTUBS_SKIP_TIMING_TESTS=$(OHHTTPSTUBS_SKIP_TIMING_TESTS) OHHTTPSTUBS_SKIP_REDIRECT_TESTS=$(OHHTTPSTUBS_SKIP_REDIRECT_TESTS)
```

Comment:
Not sure that really matters in practice in our case but shouldn't we used `$(inherited)` in that definition? (I think the default in Xcode is `GCC_PREPROCESSOR_DEFINITIONS=DEBUG=1` in debug — while it's empty in release — so not using `$(inherited)` here means that debug builds don't have `DEBUG=1` set anymore? Not that we currently use `#if DEBUG` in OHHTTPStubs as far as I know but just for configuration consistency…

Reply:
I'm honestly not sure how, but the config file looks like it's value resolves to have `DEBUG=1`, even though it's not in the file.

<img width="1223" alt="screen shot 2019-03-07 at 7 39 47 pm" src="https://user-images.githubusercontent.com/2142301/53999305-ea6ffa00-4110-11e9-9199-a46ad354e489.png">

I thought build settings always inherited from right to left in Xcode's levels view (^^ that view).. so I wouldn't expect the config file to inherit a setting from the project, but it does. 🤷‍♂️ 

Now that I think more about it - I normally don't mix project build settings and also have an xcconfig with them.  My opinion is normally that if you have an xcconfig you should put the entire value of that setting into the xcconfig instead of the project's build settings. 

@AliSoftware - What do you think of moving `DEBUG=1` to the xcconfig instead, so there is no project build setting for this one?

Reply:
Strange that it magically inherit project settings indeed 🧐
Moving DEBUG=1 in the xcconfig file makes sense, but would require two separate xcconfig files then (one for each config), just wondering if that's worth it 🤔

Reply:
Yes, probably overkill.  So do you think we should add `$(inherit)` or leave it as is?

Reply:
Let's try `$(inherited)` at least for consistency, and validate it still works as expected (eg doesn't double-define `DEBUG=1` in the resolved build settings after all)

Reply:
There were actually a few places where we had `DEBUG=1` twice, so I removed the target level preprocessor macros where needed.